docs/life_of_a_dataset.md

```diff
@@ -74,6 +78,8 @@ In the PR, please also include the validation results (`report.json` and `summar
 
 If you introduced new statistical variables, please create a Pull Request for them in the [schema repo](https://github.com/datacommonsorg/schema).
 
+After your PR is reviewed and approved, the Data Commons team will work with you to manifest the data in our internal system.
```

Comment:
s/data/"artifacts you created"/?

Reply:
This is probably important enough to be a sub-section after sending PRs?

Reply:
+1

Reply:
Created new sub-section, PTAL